enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/analytics/ChoreoAnalyticsProvider.java

```diff
@@ -59,6 +59,7 @@ public EventCategory getEventCategory() {
                 logEntry.getResponse().getResponseCodeDetails())) {
             logger.debug("Is success event");
             return EventCategory.SUCCESS;
+            // TODO: (VirajSalaka) How to handle 200 response for the websocket request. (it is a fault)
```

Comment:
This todo statement is no longer valid as those requests are already filtered out.